### PR TITLE
Measurable get started guide

### PR DESCRIPTION
**Pull request by volkergersabeck**
This could become a way to measure how many people succeed with the getting started guide.

Still missing:
- a better looking "complete" page
- opening the /complete page from Node.js worker as well. (or a different page to track who is doing Node.js vs Java)

### CODE REVIEW

File: get-started/content/quick-start/service-task.md [hunk 1]
```diff
@@ -180,6 +180,16 @@ public class ChargeCardWorker {
           Long amount = (Long) externalTask.getVariable("amount");
           LOGGER.info("Charging credit card with an amount of '" + amount + "'€ for the item '" + item + "'...");
 
+          if (Desktop.isDesktopSupported() && Desktop.getDesktop().isSupported(Desktop.Action.BROWSE)) {
```
Comment (yanavasileva) [inline get-started/content/quick-start/service-task.md]: Just a heads up:
If we make changes in the docs, we need to reflect them in the get-started repository of that example as well:
https://github.com/camunda/camunda-get-started-quickstart/blob/master/java/charge-card-worker/src/main/java/org/camunda/bpm/getstarted/chargecard/ChargeCardWorker.java
Unfortunately, this is not a trivial change, as we keep each step into separate commit (and a tag) as we offer the users to download a working project after each step. Scroll to the bottom of the page and check the greenbox (https://docs.camunda.org/get-started/quick-start/service-task/).
We also aligned the node js and java client as much as possible.

**Comment by yanavasileva** [comment]
@NPDeehan, if you need some help to get to know the getting started repositories and how the downloading of the examples for the users is working, please free to approach our team.
In case you are already aware of the whole setup, sorry for the intrusion. :)

**Comment by yanavasileva** [comment]
https://github.com/camunda/camunda-docs-static/pull/54/files#diff-b0b19b72a97a2882586e088d72df9e7eR183-R191
Just a heads up:
If we make changes in the docs, we need to reflect them in the get-started repository of that example as well:
https://github.com/camunda/camunda-get-started-quickstart/blob/master/java/charge-card-worker/src/main/java/org/camunda/bpm/getstarted/chargecard/ChargeCardWorker.java
Unfortunately, this is not a trivial change, as we keep each step into separate commit (and a tag) as we offer the users to download a working project after each step. Scroll to the bottom of the page and check the greenbox (https://docs.camunda.org/get-started/quick-start/service-task/).
We also aligned the node js and java client as much as possible.

**Comment by ThorbenLindhauer** [comment]
All done:

* Stuff is merged
* I updated the corresponding code base at https://github.com/camunda/camunda-get-started-quickstart
* I published the guide

**Pull request merged**